internal/cmd/integration-tests/utils.go

```diff
@@ -24,22 +24,42 @@ type TestLog struct {
 
 var logChan chan TestLog
 
-func executeCommand(command string, args []string, taskDescription string) {
+func executeCommand(command string, args []string, taskDescription string) bool {
 	fmt.Printf("%s...\n", taskDescription)
 	cmd := exec.Command(command, args...)
 	var stderr bytes.Buffer
 	cmd.Stderr = &stderr
+	var stdout bytes.Buffer
+	cmd.Stdout = &stdout
+
+	success := true
 	if err := cmd.Run(); err != nil {
-		log.Fatalf("Error: %s\n", stderr.String())
+		stderrStr := stderr.String()
+		if len(stderrStr) > 0 {
+			log.Fatalf("stderr: %s\n", stderrStr)
+		} else {
+			log.Fatalf("Task '%s' failed, but no stderr was printed\n", taskDescription)
+		}
```

Comment:
🤦 That's true, thank you for spotting it! I rolled back this whole chunk of code since I'm not sure if it's really necessary. And I noticed a typo in it too.